pyproject.toml

```diff
@@ -1,13 +1,20 @@
 [tool.poetry]
-name = "commodore"
-version = "v0.0.0"
+name = "syn-commodore"
+version = "v0.7.0"
```

Comment:
Please don't hardcode the version in `pyproject.toml`, but rather generate the final version from the Git tag/commit in the Github action, see https://github.com/projectsyn/commodore/blob/8671d53ef7e9fb1b613b8d0f538bb9411edb5588/Dockerfile#L25-L30 for an example.

Use https://github.com/projectsyn/commodore/blob/8671d53ef7e9fb1b613b8d0f538bb9411edb5588/Makefile#L44-L45 to generate the required values for `GITVERSION` and `PYVERSION`